spykes/django-admin-school/src/user/models.py

```diff
@@ -79,11 +114,32 @@ def __str__(self) -> str:
 
 
 class Student(AppUser):
-    dni = models.CharField(
-        verbose_name=_('dni'),
-        max_length=200
+
+    passport = models.CharField(
+        max_length=50,
+        unique=True,
+        null=True,
+        blank=True,
+        verbose_name=_('passport'),
+        help_text= _("Passport document")

```

Comment:
No need to add help_text in this case, also this field must be treated as a dni field (max_length and encryption signal)